src/test/functional/sessions.test.ts

```diff
@@ -0,0 +1,80 @@
+import { TestUtil } from "./testUtil";
+import axios from "axios";
+import chai from "chai";
+
+const testUtil = new TestUtil();
+
+const frontendURL = process.env.TEST_URL || "http://localhost:8080";
+const username = "b@a.com";
+const password = "4590fgvhbfgbDdffm3lk4j";
+const caseId = "1234";
+
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
+
+describe("/GET sessions", () => {
+  it("it should return (200) OK", async () => {
+    await testUtil.createNewUser(username, password);
+    const token = await testUtil.requestUserToken(username, password);
+
+    const headers = {
+      "Authorization": `Bearer ${token}`,
+    };
```

Comment:
setting the Authorization header seems to be repeated for most of the tests. maybe worth putting it in the beforeEach